### DISCUSSION TITLE
Where is Authenticator?

### DISCUSSION THREAD

**Discussion by JPL1** [standardnotes/forum#2831] [General] [answered]
The [Features](https://standardnotes.com/features) page states:

> Authenticator
> Use Standard Notes on both desktop and mobile as your authenticator app. Authenticator stores your 2FA secrets for other services so that you never lose them again, or have to start over when you get a new device.

I have a Professional subscription and I cannot see how to use this functionality. Or is this a use case rather than a feature?

**Comment by atmoio** [comment, marked as answer]
To use Authenticator create a new note, then set the Note Type to Authenticator.

**Comment by JPL1** [comment]
Thank you - a HowTo would be a great addition to the website btw.

**Reply by myreli** [reply]
The docs have been updated to better explain note types, including a video: https://standardnotes.com/help/77/what-are-editors
